monitor/service/api/v10.py

```diff
@@ -13,11 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import ConfigParser
```

Comment:
This is a unused import? If yes, please comment this modification in the description of this PR. This issue repeats itself through all file.